apps/ui/src/components/App/Nav.vue

```diff
@@ -63,6 +66,18 @@ const canSeeSettings = computed(() => {
 
     return admins.includes(web3.value.account.toLowerCase());
   }
+  return false;
+});
+
+const canSeeMembers = computed(() => {
+  const data = space.value?.additionalRawData;
+
+  return (
+    controller.value ||
+    (data?.admins?.length ?? 0) > 0 ||
+    (data?.moderators?.length ?? 0) > 0 ||
+    (data?.members?.length ?? 0) > 0
+  );
```

Comment:
This makes this section visible on Snapshot onchain spaces, original issue includes this:
> The page should only appear if there are members set on the space (only appear on Snapshot spaces and not SX).

Might be confusing though (Snapshot spaces = Snapshot offchain spaces, SX = Snapshot onchain spaces).

Reply:
@Sekhmet could you suggest a solution here?